<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="startRhostContent">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Target IP Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>开始扫描</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描结果为</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultOutput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ipRange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>目标主机ip</string>
    </property>
   </widget>
   <widget class="QLabel" name="portRange">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>端口范围</string>
    </property>
   </widget>
   <widget class="QLabel" name="startPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>起始端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="endPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>结束端口</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="startPortSpinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="endPortSpinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出扫描</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="prosgressLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描进度</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/gtk-go-down.png</normaloff>:/image/gtk-go-down.png</iconset>
   </property>
   <property name="text">
    <string>保存为...</string>
   </property>
   <property name="toolTip">
    <string>将输出进行保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
